applications/FluidDynamicsApplication/tests/darcy_channel_test.py

```diff
@@ -13,6 +13,7 @@ class TwoFluidNoRedistanceSolver(two_fluids_solver.NavierStokesTwoFluidsSolver):
     """
     def __init__(self, model, settings):
         super(TwoFluidNoRedistanceSolver,self).__init__(model,settings)
+        self._reinitialization_type = "none"
```

Comment:
Couldn't we add this to the .json settings file in the TwoFluidDarcyTest folder?

Reply:
> Couldn't we add this to the .json settings file in the TwoFluidDarcyTest folder?

Done.